Makefile

```diff
@@ -1009,14 +1009,14 @@ GO_SOURCES := $(GO_PROTOS:%.proto=%.pb.go)
 PBJS := $(NODE_RUN) $(UI_ROOT)/node_modules/.bin/pbjs
 PBTS := $(NODE_RUN) $(UI_ROOT)/node_modules/.bin/pbts
 
-JS_PROTOS_CCL := $(filter %/ccl/storageccl/engineccl/enginepbccl/key_registry.proto %/ccl/storageccl/engineccl/enginepbccl/stats.proto,$(GO_PROTOS))
+JS_PROTOS_CCL := $(filter %/ccl/storageccl/engineccl/enginepbccl/stats.proto,$(GO_PROTOS))
```

Comment:
The C++ version of protos needs an explicit list, so the different behavior probably calls for a comment.

Reply:
Done.